xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.h

```diff
@@ -118,7 +118,7 @@ namespace Shaders {
     GLint m_hYTex;
     GLint m_hUTex;
     GLint m_hVTex;
-    GLint m_hMatrix;
+    GLint m_hMatrix = -1;
```

Comment:
to match the others: https://github.com/xbmc/xbmc/blob/8405bbf242a73e7b88641c7db4a9808fd46a2f47/xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/YUV2RGBShader.cpp#L185